src/document/prompts.js

```diff
@@ -0,0 +1,19 @@
+import { Template } from '../template/Template.js';
+
+export const learnCSS = new Template(
+  ['html', 'template', 'format'],
+  `Given some HTML, give me the CSS selector to select all the elements related to what the user is scraping. Do not use pseudo-elements, selectors should be compatible with css-select.
```

Comment:
curious, what if we make an alternate prompt that asks it to write code? Someting like

```Given some HTML, give me javascript code that boils down the page to just the content relevant to the scraping task below....```

For example see https://github.com/fetchfox/fetchfox/blob/x/instr-code/src/fetch/CodeInstructions.js and https://github.com/fetchfox/fetchfox/blob/x/instr-code/src/fetch/prompts.js#L147

Reply:
basically I'm worried about making something that's too specialized or specific. The current pagination in Instructions.js is kinda like that. o3-mini is pretty good at writing small snippets of code.